pod/perlexperiment.pod

```diff
@@ -469,6 +453,52 @@ Removed in Perl 5.36.0
 
 =back
 
+=head2 Features abandoned in place
+
+These are experiments that are considered failures, but due to mass
+use have been left in place.
+
+This will likely only ever be the switch and smartmatch features,
+which were introduced before we had a well-defined experimental
+feature process.
```

Comment:
The wording of this paragraph -- particularly the first sentence ( `likely only ever be`) -- is awkward.  Perhaps something like:
```
The switch and smartmatch feaatures are likely to be the only ones which 
will ever fall into this category.  They were introduced before we had a 
well-defined experimental feature process.
```

Reply:
Updated